src/sherpa_ai/error_handling/agent_error_handler.py

```diff
@@ -11,15 +11,14 @@ class AgentErrorHandler:
 
     def __init__(self):
         self.error_map = {
-            openai.error.APIError: "OpenAI API returned an API Error",
-            openai.error.APIConnectionError: "Failed to connect to OpenAI API",
-            openai.error.RateLimitError: "OpenAI API request exceeded rate limit",
-            openai.error.AuthenticationError: (
-                "OpenAI API failed authentication or incorrect token"
-            ),
-            openai.error.Timeout: "OpenAI API Timeout error",
-            openai.error.ServiceUnavailableError: "OpenAI API Service unavailable",
-            openai.error.InvalidRequestError: "OpenAI API invalid request error",
+            # openai.APIError: "OpenAI API returned an API Error",
```

Comment:
Can we find the new errors for openai? if not, maybe we should keep it as an empty map

Reply:
If we haven't find all new errors from OpenAI yet, please add a TODO comment here or create an issue so that we can revisit this point.

Reply:
sure https://linear.app/aggregate-intellect/issue/SHX-100